20_2 TreeList Column Fixing: rewrite (#796)
DevExpress/devextreme-demos: JSDemos/Demos/TreeList/ColumnFixing/description.md

> When the width of all columns exceeds the component's width, horizontal scrolling occurs. TreeList allows to fix certain columns to the right or left so they would always be visible on scroll. To enable this feature, set **columnFixing**.[enabled](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columnFixing/#enabled) to **true**.

Reviewer: ```suggestion
When the width of all columns exceeds the component's width, horizontal scrolling occurs. TreeList allows users to fix individual columns to the right or left so they would remain visible on scroll. To enable this feature, set the **columnFixing**.[enabled](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columnFixing/#enabled) property to **true**.
```

> When the width of all columns exceeds the component's width, horizontal scrolling occurs. TreeList allows users to fix individual columns to the right or left so they would remain visible on scroll. To enable this feature, set the **columnFixing**.[enabled](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columnFixing/#enabled) property to **true**.

Reviewer 2: ```suggestion
A horizontal scrollbar appears when the sum width of all columns exceeds the TreeList component's width. The component allows users to fix individual columns at the right or left border so they would remain always visible. To enable this feature, set the **columnFixing**.[enabled](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columnFixing/#enabled) property to **true**.
```

> A horizontal scrollbar appears when the sum width of all columns exceeds the TreeList component's width. The component allows users to fix individual columns at the right or left border so they would remain always visible. To enable this feature, set the **columnFixing**.[enabled](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columnFixing/#enabled) property to **true**.

Reviewer 3: ```suggestion
A horizontal scrollbar appears when the total width of all columns exceeds the TreeList component's width. The component allows users to fix individual columns at the right or left border so that they are always visible. To enable this feature, set the **columnFixing**.[enabled](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columnFixing/#enabled) property to **true**.
```

> To fix/unfix a column, users should right-click the column's header and select the desired operation in the appeared context menu. If a column should never be fixed or unfixed, set its [allowFixing](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#allowFixing) property to **false**.

Reviewer 2: ```suggestion
To fix or unfix a column, users should right-click the column's header and select the corresponding operation in the context menu. To forbid users to fix or unfix a column, set its [allowFixing](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#allowFixing) property to **false**.
```

Author: "**_To_** forbid users **_to_** fix..." - have you considered the double "to"?

Reviewer 2: I had indeed. There are many ways to express this idea but each has its own disadvantages.

> To fix or unfix a column, users should right-click the column's header and select the corresponding operation in the context menu. To forbid users to fix or unfix a column, set its [allowFixing](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#allowFixing) property to **false**.

Reviewer 3: ```suggestion
To fix or unfix a column, users should right-click the column's header and select the corresponding operation in the context menu. To prevent users from fixing or unfixing a column, set its [allowFixing](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#allowFixing) property to **false**.
```

> To fix/unfix a column in the UI, users should right-click the column's header to open the context menu and choose the desired operation. If some column should never be fixed or unfixed, set its [allowFixing](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#allowFixing) to **false**.

Reviewer: We are talking about UI in the first paragraph. It would be logical to move this paragraph up.

```suggestion
To fix/unfix a column, users should right-click the column's header and select the desired operation in the appeared context menu. If a column should never be fixed or unfixed, set its [allowFixing](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#allowFixing) property to **false**.
```

> To fix a column programmatically, set its [fixed](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#fixed) property to **true**. The default position is TreeList's left, but you can set the column's [fixedPosition](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#fixedPosition) property to *"right"*.

Reviewer 2: TreeList's left what?

```suggestion
A fixed column attaches to the TreeList's left border or last fixed column there but you can move the column to the right border. Set the column's [fixedPosition](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#fixedPosition) property to *"right"*.
```

Author: There's no [fixed] prop. in the suggestion.
>TreeList's left what?
```suggestion
To fix a column programmatically, set its [fixed](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#fixed) property to **true**. The default position is the TreeList's left, but you can set the column's [fixedPosition](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#fixedPosition) property to *"right"*.
```

Reviewer 2: Either way, I still do not understand what "the TreeList's left" is. Is it:
- The tree list's left border.
- To the left of the tree list's scrollable area.
?
The phrase itself (the tree list's left) is really strange.

Author: It can't be "the left of the TreeList's scrollable area" because if it were, the fixedPosition poroperty would make no sense.
